src/containers/shared/components/Transaction/OfferCreate/TableDetail.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Amount } from '../../Amount';
+
+export const TableDetail = (props: any) => {
+  const { t } = useTranslation();
+  const { instructions } = props;
+  const { gets, pays, price, pair, cancel } = instructions;
+
+  return pays && gets ? (
+    <div className="offercreate">
```

Comment:
There has to be some way to simplify this code/make reusable components that all the other txs can use for their table detail

Reply:
I have not really noticed much in the way of commonality.  I will look harder at this.

Reply:
I don't think the code looks very similar, but in the deployed version they look very similar